4_Zadania_dodatkowe/6.InfiniteSlider/js/app.js

```diff
@@ -0,0 +1,71 @@
+$(document).ready(function () {
+   // console.log('test');
+
+    var elementPrevButton = $('#prevPicture');
+    var elementNextButton = $('#nextPicture');
+    // console.log(elementPrevButton, elementNextButton);
+
+    var elementListLi = $('li');
+    // console.log(elementListLi);
+
+    var elementUl = $('ul');
```

Comment:
I noticed you prefixed above variables with `element`. Nothing wrong with it but more common way to say "hey, this is an element" is to start variable name with `$` (`$nextButton` for example).